projects/angular/src/forms/combobox/combobox.ts

```diff
@@ -194,7 +194,14 @@ export class ClrCombobox<T>
 
   @Input('clrMulti')
   set multiSelect(value: boolean | string) {
-    if (value) {
+    let isMultiSelectModal = false;
```

Comment:
I'm not sure in what situation you would be passing a string here. This is meant to be a boolean, and generally its good to restrict typing in these cases. It can lead to confusing outcomes for consumers if they think a string can be passed like `"0"`, is sending a boolean really a problem?

Regardless, this is a lot of logic. We can just coerce the value into a boolean and not try to handle individual type cases.

```js
value = !!value;
```

Reply:
On our website the usage is documented as:
> The multi selection mode is triggered with a clrMulti="true" 

This implies that `clrMulti="false"` should also work.
And I don't thing that the coercion will not work with the case of `clrMulti="false"`

Reply:
This would be better checking against more explicit values. There's a lot of unnecessary `typeof` checks here that are only necessary because we are duck typing the value.

So something more like...

```
let isMultiSelectModal: boolean;

switch (true) {
  case (value === 'false' || value === false):
     isMultiSelectModal = false;
     break;
  case (value === 'true' || value === true):
     isMultiSelectModal = true;
     break;
   default:
     isMultiSelectModal = false;
}
```

... then you could finish off with something like...

```
this.optionSelectionService.selectionModel = isMultiSelectModal ? new MultiSelectComboboxModel<T>() : new SingleSelectComboboxModel<T>();
```

...narrowing down a lot of conditional code here.

Reply:
Please check the comments from Bozhidar Dryanovski in issue #6432 :
`The reason for having string as a type is that the strict Angular (fullTemplateTypeCheck & strictTemplates) check requires every field that is boolean to have an additional type of string.`

So here we have the fix to support string value 'true' and 'false'.

@mathisscott , addressed the comments.